tests/antithesis/test-template/go-delete-keys/serial_driver_delete_keys.go

```diff
@@ -45,77 +49,64 @@ func Connect() *client.RecordingClient {
 	return cli
 }
 
-func DeleteKeys() {
-	// This function will:
-	// 1. Get all keys
-	// 2. Select half of the keys received
-	// 3. Attempt to delete the keys selected
-	// 4. Check that the keys were deleted
-
-	ctx := context.Background()
-
-	// Connect to an etcd node
-	cli := Connect()
-
-	// Get all keys
-	resp, err := cli.Get(ctx, "", clientv3.WithPrefix())
-
-	// Antithesis Assertion: sometimes get with prefix requests are successful. A failed request is OK since we expect them to happen.
-	assert.Sometimes(err == nil, "Client can make successful get all requests", map[string]any{"error": err})
-	cli.Close()
-
-	if err != nil {
-		log.Printf("Client failed to get all keys: %v", err)
-		os.Exit(0)
-	}
-
-	// Choose half of the keys
-	var keys []string
-	for _, k := range resp.Kvs {
-		keys = append(keys, string(k.Key))
-	}
-	half := len(keys) / 2
-	halfKeys := keys[:half]
+var etcdTraffic traffic.Traffic = traffic.EtcdTraffic{
+	KeyCount: 10,
+	// Please keep the sum of weights equal 100.
+	Requests: []robustnessrand.ChoiceWeight[traffic.EtcdRequestType]{
+		{Choice: traffic.Get, Weight: 50},
+		{Choice: traffic.Put, Weight: 50},
+	},
+}
 
-	// Connect to a new etcd node
-	cli = Connect()
+var profile = traffic.Profile{
+	MinimalQPS:                     100,
+	MaximalQPS:                     1000,
+	BurstableQPS:                   1000,
+	ClientCount:                    3,
+	MaxNonUniqueRequestConcurrency: 3,
+}
 
-	// Delete half of the keys chosen
-	var deletedKeys []string
-	for _, k := range halfKeys {
-		_, err := cli.Delete(ctx, k)
-		// Antithesis Assertion: sometimes delete requests are successful. A failed request is OK since we expect them to happen.
-		assert.Sometimes(err == nil, "Client can make successful delete requests", map[string]any{"error": err})
-		if err != nil {
-			log.Printf("Failed to delete key %s: %v", k, err)
-		} else {
-			log.Printf("Successfully deleted key %v", k)
-			deletedKeys = append(deletedKeys, k)
+// wrap converts a receive-only channel to receive-only struct{} channel
+func wrap[T any](from <-chan T) <-chan struct{} {
```

Comment:
Note on style. Not blocking this PR. As we are not writing C, there is no limit on order of functions. Best to write them in order of reading. From most important to least important. From top function in hierarchy to function at the bottom. So `wrap` should be after `testRobustness` and `testRobustness` should be after `main`.

Reply:
Roger that.  Will keep that in mind for future PRs.